mappings/net/minecraft/client/gui/widget/WidgetListVideoSettings.mapping

```diff
@@ -6,11 +6,15 @@ CLASS cox net/minecraft/client/gui/widget/WidgetListVideoSettings
 		METHOD a drawEntry (IIIIZF)V
 		METHOD mouseClicked (DDI)Z
 			ARG 0 mouseX
-			ARG 1 mouseY
```

Comment:
I'm really not sure what's going on here - it's like one tool was started at 0, and another at 1. Similarly why there's a `keyCode` argument here at all.

I'm happy to clean this up, just would like someone else to look over it first.